README.md

```diff
@@ -47,7 +47,7 @@ buildscript {
 
     dependencies {
     	// ...
-        classpath 'com.github.triplet.gradle:play-publisher:1.2.1'
+        classpath 'com.github.triplet.gradle:play-publisher:1.2.2-beta1'
```

Comment:
Agreed. Also, I think we should start development on v2.0 or v1.3 so we can bump the AGP version, meaning 1.2.2 won't be correct.

Reply:
I'm ok with both of those thoughts. I suggest that the Kotlin version be 1.2.3, then we can branch that to start working on v2.0 using the latest AGP version (including Bundle prep)

Because, if we don't merge this back to master, PRs incoming will continue to be on the Groovy files. :/

Reply:
I agree with you that this PR needs to be merged before we start working on v2.0, but I don't think we should release v1.2.2: if the release is just moving to Kotlin, we'll be adding bugs, not removing them. 😂